Add third party library (#2010)
pmndrs/zustand: docs/integrations/third-party-libraries.md

- [zustand-rx](https://github.com/patdx/zustand-rx) — A Zustand middleware enabling you to subscribe to a store as an RxJS Observable.
- [zustand-saga](https://github.com/Nowsta/zustand-saga) — A Zustand middleware for redux-saga (minus redux).
- [zustand-store-addons](https://github.com/Diablow/zustand-store-addons) — React state management addons for Zustand.

> - [zustand-sync-tabs](https://www.npmjs.com/package/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)

Reviewer: Do you have a Github link? It was hard to found quickly from the NPM page, at least for me.

Most if not all links here are repository links, so I think at this point this is what most people expect.

> - [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)

Reviewer: You can add a full stop at the end and rephrase "SameOrigin" to something like "for apps from the same origin."

```suggestion
- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin).
```

> - [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin).

Reviewer: Just like in a previous PR:

```suggestion
- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs/windows/iframes with same origin.
```

> - [zustand-sync-tabs](https://www.npmjs.com/package/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin) [GitHub](https://github.com/mayank1513/zustand-sync-tabs)

Reviewer 2: Can we do this?
```suggestion
- [zustand-sync-tabs](https://github.com/mayank1513/zustand-sync-tabs) — Zustand middleware to easily sync Zustand state between tabs / windows / iframes (SameOrigin)
```
